src/helpers/Workspace.cpp

```diff
@@ -58,7 +58,46 @@ CWorkspace::~CWorkspace() {
 void CWorkspace::startAnim(bool in, bool left, bool instant) {
     const auto ANIMSTYLE = m_fAlpha.m_pConfig->pValues->internalStyle;
 
-    if (ANIMSTYLE == "fade") {
+    if (ANIMSTYLE.find("slidefade") == 0) {
+        const auto PMONITOR = g_pCompositor->getMonitorFromID(m_iMonitorID);
+        float      movePerc  = 100.f;
+
+        if (ANIMSTYLE.find("%") != std::string::npos) {
+            try {
+                auto percstr = ANIMSTYLE.substr(ANIMSTYLE.find_last_of(' '));
```

Comment:
findlastof + 1

Reply:
https://github.com/hyprwm/Hyprland/pull/3008/commits/9679a745041bf1e7637a1990619f6ce549dba8f5

btw i noticed this is the same for the percentage check of the `popin` window anim style too